SU-SWS/decanter-react
# ADAPT-000: Bundlers and ESModule Classnames alternative

# READY FOR REVIEW

# Summary
- Upgrades packages
- Swaps `classnames` for `clsx` as classnames was causing issues with bundlers and tests
- Adds `npm run package` to compile new packages
- Removes broken Jest tests for the time being

# Needed By (Date)
- End of sprint

# Urgency
- Med/low
- 
# Steps to Test

1. Check out storybook preview build
2. Review code
3. Checkout branch locally and run `nvm use && npm install`
3. Pull into your latest work in https://github.com/SU-SWS/ood_online_giving/pull/11 and rebuild
4. Ensure nothing breaks in https://github.com/SU-SWS/ood_online_giving/pull/11 after running `npm update && npm run develop`
5. Run jest tests `npm run test`
6. Run `npm run build`
7. Run `npm run dev`
8. Run `npm run package`
